ui/components/app/assets/nfts/nfts-items/nfts-items.js

```diff
@@ -242,13 +241,14 @@ export default function NftsItems({
                 collectionImage={collectionImage}
                 collectionName={collectionName}
               />
-              <Typography
+              <Text
                 color={Color.textDefault}
-                variant={TypographyVariant.H5}
+                variant={TextVariant.bodyMd}
                 margin={2}
+                as="h5"
```

Comment:
Currently `TypographyVariant.H5` renders as an `h5` so adding this here to retain html structure and pass unit tests